vyxal/elements.py

```diff
@@ -2564,17 +2581,10 @@ def polynomial_expr_from_coeffs(lhs, ctx):
     ts = vy_type(lhs)
     x = sympy.symbols("x")
     return {
-        NUMBER_TYPE: lambda: str(
-            sum(map(lambda arg: x ** arg, range(0, lhs + 1)))
-        ),
+        NUMBER_TYPE: lambda: str(sum(x ** arg for arg in range(0, lhs + 1))),
         str: lambda: lhs,
         list: lambda: str(
-            sum(
-                map(
-                    lambda arg: arg[1] * x ** arg[0],
-                    enumerate(reverse(lhs, ctx)),
-                )
-            )
+            sum(arg[1] * x ** arg[0] for arg in enumerate(reverse(lhs, ctx)))
```

Comment:
now you're not using a lambda, you can eliminate the tuple indexing:
```suggestion
            sum(c * x ** i for i, c in enumerate(reverse(lhs, ctx)))
```